docs/source/tutorials/datasets.rst

```diff
@@ -225,6 +225,16 @@ Adding data from a private cloud
         2. A way to enable cloud integrations from the SDK?
         3. Sample outputs of the given script
 
+.. note::
+
+    We strongly encourage you to follow and adapt the below script for a safe way to upload data to your dataset.
```

Comment:
```suggestion
    We strongly encourage you to follow and adapt the below script for an idiomatic way to upload data to your dataset.
```
Replace safe with idiomatic. Let's not ask people why it might not be 'safe' :D